doc/build-windows.md

```diff
@@ -94,6 +98,18 @@ Ubuntu Zesty 17.04 <sup>[2](#footnote2)</sup>:
 
 Once the tool chain is installed the build steps are common:
 
+Note that for WSL the Bitcoin Core source path MUST be somewhere in the default mount file system, for
```

Comment:
This should also be written under building for 32-bit windows.

Reply:
Done.